test/Serialization/Address.purs

```diff
@@ -125,7 +91,7 @@ enterpriseAddressFunctionsTest = do
   assertTrue_ $ enterpriseAddressPaymentCred eaddr == keyHashCredential pkh
 
 pointerAddressFunctionsTest :: TestPlanM Unit
-pointerAddressFunctionsTest = do
+pointerAddressFunctionsTest = group "PointerAddress tests" $ do
```

Comment:
This does not produce any output when the test suite is run, right? I believe it's required to use `test` at least once in a group in order for output to appear.